src/main/scala/edu/ie3/powerFactory2psdm/config/ConversionConfig.scala

```diff
@@ -7,115 +7,34 @@
 package edu.ie3.powerFactory2psdm.config
 
 import edu.ie3.powerFactory2psdm.config.ConversionConfig.StatGenModelConfigs
+import edu.ie3.powerFactory2psdm.config.ConversionConfigUtils.ParameterSource
+import edu.ie3.powerFactory2psdm.config.model.{
+  PvConversionConfig,
+  WecConversionConfig
+}
 
 final case class ConversionConfig(modelConfigs: StatGenModelConfigs)
 
 /** Config used for the grid conversion
   */
 object ConversionConfig {
 
-  /** Groups different sources for certain parameters of power factory models
-    */
-  sealed trait ParameterSource
-
-  /** Take values from the load flow (Lastfluss) specification of the model
-    */
-  final case object LoadFlowSource extends ParameterSource
-
-  /** Take values from the basic data (Basisdaten) specification of the model
-    */
-  final case object BasicDataSource extends ParameterSource
-
   /** Groups all configs for model conversion of static generators.
     *
     * @param pvConfig
     *   config for the pv models
     * @param sRatedSource
-    *   which apparent power source to choose from the PowerFactory model
+    *   which apparent powegit logr source to choose from the PowerFactory model
```

Comment:
pls fix this - dunno what `powegit logr` is ...

Reply:
me neither 😂